operator/internal/validation/openshift/alertingrule_test.go

```diff
@@ -206,6 +257,151 @@ func TestAlertingRuleValidator(t *testing.T) {
 				},
 			},
 		},
+		{
+			desc: "missing severity label",
+			spec: &lokiv1beta1.AlertingRule{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "alerting-rule",
+					Namespace: "example",
+				},
+				Spec: lokiv1beta1.AlertingRuleSpec{
+					TenantID: "application",
+					Groups: []*lokiv1beta1.AlertingRuleGroup{
+						{
+							Rules: []*lokiv1beta1.AlertingRuleGroupSpec{
+								{
+									Expr: `sum(rate({kubernetes_namespace_name="example", level="error"}[5m])) by (job) > 0.1`,
+									Annotations: map[string]string{
+										summaryLabelname:     "alert summary",
+										descriptionLabelName: "alert description",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			wantErrors: []*field.Error{
+				{
+					Type:     field.ErrorTypeInvalid,
+					Field:    "Spec.Groups[0].Rules[0].Labels",
+					BadValue: nilMap,
```

Comment:
I think you can just put `nil` in here instead of a `nilMap` map variable that is null

Reply:
My bad, didn't see that was an interface.